lib/src/components/NxCombobox/__tests__/NxCombobox.test.tsx

```diff
@@ -391,6 +441,46 @@ describe('NxCombobox', function() {
       expect(inputElement).toHaveAttribute('aria-activedescendant', optionBtns[0].id);
     });
 
+    it('sets aria-expanded to false when a dropdown item is selected by Enter key', async function() {
+      const user = userEvent.setup(),
+          { getByRole } = quickRender({ matches: [{ id: '1', displayName: 'Foo' }] }),
+          inputElement = getByRole('combobox'),
+          optionBtn = getByRole('option');
+
+      // initial focus of the input
+      await user.keyboard('[Tab]');
+      expect(inputElement).toHaveAttribute('aria-expanded', 'true');
+
+      // selection is made with Enter key
+      await user.keyboard('[ArrowDown]');
+      expect(optionBtn).toHaveAttribute('aria-selected', 'true');
+      await user.keyboard('[Enter]');
+      expect(inputElement).toHaveAttribute('aria-expanded', 'false');
+    });
+
+    it('sets aria-expanded to true when the input receives focus after dropdown selection and blur', async function() {
+      const user = userEvent.setup(),
+          { getByRole } = quickRender({ matches: [{ id: '1', displayName: 'Foo' }] }),
+          inputElement = getByRole('combobox');
+
+      await user.keyboard('[Tab]');
```

Comment:
See https://testing-library.com/docs/user-event/convenience#tab

Reply:
> See https://testing-library.com/docs/user-event/convenience#tab

Thanks for forwarding the link. Fixed here: [f85b73a](https://github.com/sonatype/sonatype-react-shared-components/pull/876/commits/f85b73a035d3ea670ef008a821346c94d3d7e53f)